### DISCUSSION TITLE
Deleting folders or renaming is currently not possible with 6.1.1 if Music App is enabled

### DISCUSSION THREAD

**Discussion by frederikb96** [nextcloud/all-in-one#2767] [Questions] [answered]
### Steps to reproduce
0. Have music app enabled
1. Create a folder
2. Delete the folder
3. Error, folder does not get deleted

### Expected behavior 
Folder gets deleted

### Actual behavior 
Error, folder does not get deleted

### Background
See this [post for information](https://help.nextcloud.com/t/error-deleting-folders-or-moving-folders/162994/7)

I just wanted to let others know, that this is a problem currently. I was wondering for some days now, why it is not possible to delete folders anymore :)

### Host OS 
Ubuntu 22.04

#### Nextcloud AIO version 
6.1.1

#### Current channel 
stable

**Comment by szaimen** [comment, marked as answer]
Duplicate of https://github.com/owncloud/music/issues/1068

**Comment by frederikb96** [comment]
Thanks to the update this night to 26.0.4, this is fixed now in the stable branch
